Microsoft.Toolkit.HighPerformance/Enumerables/ReadOnlyRefEnumerable{T}.cs

```diff
@@ -121,6 +120,63 @@ internal ReadOnlyRefEnumerable(object? instance, IntPtr offset, int length, int
         }
 #endif
 
+        /// <summary>
+        /// Gets the total available length for the sequence.
+        /// </summary>
+        public int Length
+        {
+            [MethodImpl(MethodImplOptions.AggressiveInlining)]
+#if SPAN_RUNTIME_SUPPORT
+            get => this.span.Length;
+#else
+            get => this.length;
+#endif
+        }
+
+        /// <summary>
+        /// Gets the element at the specified zero-based index.
+        /// </summary>
+        /// <returns>A reference to the element at the specified index.</returns>
```

Comment:
Nit: this is missing the XML node for `index`. Same for all the other properties below 🙂

Reply:
Fixed in the latest commit 👍